.cirrus.yml

```diff
@@ -75,8 +75,8 @@ task:
         $ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/pub get
         cd $ENGINE_PATH/src/flutter/lib/web_ui
         $ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/pub get
-        export DART="$ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/dart"
-        $DART dev/firefox_installer_test.dart
+        export FELT="$ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/dart dev/felt.dart"
+        $FELT test --browser=firefox --firefox-version=71.0
```

Comment:
If we are going to specify the version here, why do we also have it in `browser_lock.yaml`?

Reply:
Done.

I personally imagined lock file as the place we put the defaults. when the developers only run felt --browser=firefox they will fetch the locked version. However the tools allows them to use whatever version they want to try.

Reply:
It's useful to have a single source of truth for browser versions. Otherwise when you test on your local workstation you'd be using one version, and Cirrus would be using a different version, which could introduce test result differences between local and Cirrus builds.

Reply:
I changed it.